packages/sources/nav-consulting/src/endpoint/reserve.ts

```diff
@@ -0,0 +1,34 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import { navConsultingTransport } from '../transport/reserve'
+
+export const inputParameters = new InputParameters(
+  {
+    fund: {
+      required: true,
+      type: 'string',
+      description:
+        'Name of the fund, used to select {$fund}_API_KEY {$fund}_SECRET_KEY from env variables',
```

Comment:
[low priority] just noticed this. Should it be `${fund}` instead of the reverse `{$`?
also applies to anchorage, coinbase-prime, and harris-and-trotter EAs